dali/pipeline/data/types.cc

```diff
@@ -88,6 +88,9 @@ void TypeInfo::Copy(void *dst,
     const void *src, Index n, cudaStream_t stream, bool use_copy_kernel) const {
   constexpr bool is_host_to_host = std::is_same<DstBackend, CPUBackend>::value &&
                                    std::is_same<SrcBackend, CPUBackend>::value;
+  if (n == 0)
```

Comment:
DO we want this now?

Reply:
I think so, a copy of 0 elements is no-op

Reply:
I was wondering if we want to make it noop, or fail. I'm fine both ways.